python-checks/src/test/java/org/sonar/python/checks/UnusedLocalVariableCheckTest.java

```diff
@@ -35,4 +36,19 @@ public void custom() {
     check.format = "(_|myignore)";
     PythonCheckVerifier.verify("src/test/resources/checks/unusedLocalVariableCustom.py", check);
   }
+
+  @Test
+  public void quickFixTest() {
```

Comment:
_minor_ Test naming is all over the place in the code base, but we should try to stick to `snake_case`, and maybe indicate that the quick fix is only for sequence unpacking.
```suggestion
  public void tuple_quick_fix() {
```